### ISSUE TITLE
Bug in V1.0 * corpora (and in schema): bad org/@role values

### ISSUE THREAD

**Issue by TomazErjavec** [clarin-eric/ParlaMint#9]
If #8 is to be implemented, then `org/@role` needs to be correced too - we now have `political_party` and `ethnic_communities` as possible values. They should be changed to `politicalParty` and `ethnicCommunity" (note change to singular as well!).

**Comment by matyaskopp** [comment]
## (CZ) list of new (or camelCased existing ones - with star) org roles with number of occurrences in corpus:
```
    299 interparliamentaryFriendshipGroup
    280 subcommittee
    114 committee
     77 commission
     44 politicalGroup
     31 delegation
     13 senate
*    11 politicalParty
      9 institution
      7 supervisoryBoard
      5 workingGroup
      3 czechNationalCouncil
      3 chamberOfThePeople
      3 chamberOfTheNations
      2 europeanParliament
      1 president
      1 internationalOrganizations
      1 boardOfDirectors
```

**Comment by TomazErjavec** [comment]
In c99c006 I added the values used by CZ to the , along with comments into ParlaMint/Schema/ParlaMint-teiCorpus.rng (i.e.  https://github.com/clarin-eric/ParlaMint/blob/main/Schema/ParlaMint-teiCorpus.rng). More about this for CZ in #15.

**Comment by TomazErjavec** [comment]
Fixed in 393e208, closing.
